lib/tmuxinator/deprecations.rb

```diff
@@ -1,19 +1,12 @@
 module Tmuxinator
   module Deprecations
-    def rvm?
-      yaml["rvm"]
-    end
-
-    def rbenv?
-      yaml["rbenv"]
-    end
-
-    def pre_tab?
-      yaml["pre_tab"]
-    end
-
-    def cli_args?
-      yaml["cli_args"]
+    def method_missing(name, *args, &block)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used.<br>Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.